### ISSUE TITLE
Audit on-site notifications

### ISSUE THREAD

**Issue by humitos** [readthedocs/ext-theme#27]
We want to audit all the on-site notifications to confirm their copy makes sense and they are useful. Basically, we want to confirm if a user that reads the notification and clicks on the link to "solve the problem" has all the context required on the linked page to be able to solve it without requiring jumping into the Read the Docs documentation.

A good example of a not useful notification is the notification for a failure webhook attached:

> The project {{ project.name }} doesn't have a valid webhook set up, commits won't trigger new builds for this project. See <a href='{{ url_integrations }}'>the project integrations</a> for more information.

(from https://github.com/readthedocs/readthedocs.org/blob/b68a92106b5926c7e708f77aa3d04e2f30b0a24b/readthedocs/oauth/notifications.py#L44-L48)

that links to the `Admin -> Integrations` page that has no context. Example: https://readthedocs.com/dashboard/readthedocs-ops/integrations/

Original card at: https://trello.com/c/tY0Mi8v4/771-audit-all-trial-subscription-emails-notifications-to-ensure-they-work-and-are-useful

**Comment by humitos** [comment]
I don't have any immediate actionable work here but I assume there are a bunch of notifications with not clear enough instructions about how to solve the problem --as the one showed in the description, which it has its own issue now at #28 

I'd say the work required here is:

1. open all the `notifications.py` and `messages.py` files
2. read each of the notifications, follow the link and double check the user has all the information required to solve the problem

If 2) fails, we should open an issue per notification not passing that step --or fix it immediately if it's easy.

**Comment by agjohnson** [comment]
Makes sense. I think this is probably a backend change then, so I'll remove it from this roadmap and it might make sense to move this off this repo too.
